### PR TITLE
Features/docker

### PR DESCRIPTION
**Pull request by derf974**
I added Dockerfile for building the app by docker.

For building the applications , you have to:

docker build -t easy-admin-demo .

For running: docker run -d  -p 8000:8000 easy-admin-demo

Sorry for my english.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+WORKDIR /app
+
+RUN apk add libzip-dev bash 
+
+RUN wget https://get.symfony.com/cli/installer -O - | bash
+
+ENV PATH="$HOME/.symfony/bin:$PATH"
+
+RUN docker-php-ext-install zip
```
Comment (IonBazan) [inline Dockerfile:15]: I think you should also install `pdo_sqlite`

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,21 @@
+FROM php:zts-alpine
+
+COPY . /app/
+
+RUN cd /usr/bin/ && wget https://raw.githubusercontent.com/composer/getcomposer.org/76a7060ccb93902cd7576b67264ad91c8a2700e2/web/installer -O - -q | php -- --quiet
```
Comment (IonBazan) [inline Dockerfile:5]:
```suggestion
RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
```

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,21 @@
+FROM php:zts-alpine
+
+COPY . /app/
```
Comment (IonBazan) [inline Dockerfile:3]: Move this to the very end of the Dockerfile

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+ENV PATH="$HOME/.symfony/bin:$PATH"
+
+RUN docker-php-ext-install zip
+
+RUN php -d memory_limit=-1 /usr/bin/composer.phar require doctrine/annotations
+
+RUN php -d memory_limit=-1 /usr/bin/composer.phar install
+
+ENTRYPOINT php bin/console server:run 0.0.0.0:8000
```
Comment (IonBazan) [inline Dockerfile:21]:
```suggestion
ENTRYPOINT symfony serve --no-tls
```

File: Dockerfile [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+RUN wget https://get.symfony.com/cli/installer -O - | bash
+
+ENV PATH="$HOME/.symfony/bin:$PATH"
+
+RUN docker-php-ext-install zip
+
+RUN php -d memory_limit=-1 /usr/bin/composer.phar require doctrine/annotations
+
+RUN php -d memory_limit=-1 /usr/bin/composer.phar install
```
Comment (IonBazan) [inline Dockerfile:19]:
```suggestion
RUN composer install --no-dev -n
```

**Comment by javiereguiluz** [comment]
Thanks for submitting this Pull Request. However, we must close it without merging because we're discontinuing this project and archiving this repository in favor of the new EasyAdmin Demo project.

More details: https://easycorp.github.io/blog/posts/a-new-easyadmin-demo-application

**Pull request closed without merging**